docker-cloud/builds/index.md

```diff
@@ -2,16 +2,10 @@
 description: Manage Builds and Images in Docker Cloud
 keywords:
 - builds, images, Cloud
-menu:
-  main:
-    parent: builds
-    weight: -80
 notoc: true
-title: Builds and Images Overview
+title: Build and images overview
```

Comment:
This should remain plural because you can have multiple builds. Also, as a landing/index page, it feels weird to decap Overview for some reason. 🤔